### PR TITLE
Allow issue reprioritisation and commenting, and fetching of issues and comments

### PR DESCRIPTION
**Pull request by rjkip**
* [ ] Verify reprioritisation works (pending EDIT_ISSUE rights on JIRA)

### CODE REVIEW

**Comment by rjkip** [comment]
Ping @arothuis

File: src/Surfnet/Conext/OperationsSupportBundle/Console/Command/JiraMyPermissionsCommand.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+namespace Surfnet\Conext\OperationsSupportBundle\Console\Command;
+
+use Jira_Api as JiraApiClient;
+use Surfnet\JiraApiClientBundle\ApiClient;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\VarDumper\VarDumper;
+
+final class JiraMyPermissionsCommand extends ContainerAwareCommand
```
Comment (arothuis) [inline src/Surfnet/Conext/OperationsSupportBundle/Console/Command/JiraMyPermissionsCommand.php]: Nit: could have a more descriptive name.

**Comment by arothuis** [comment]
Maybe fix the name, other than that: :rocket:

File: src/Surfnet/Conext/OperationsSupportBundle/Console/Command/JiraMyPermissionsCommand.php (hunk 1 above)
Comment (rjkip) [inline src/Surfnet/Conext/OperationsSupportBundle/Console/Command/JiraMyPermissionsCommand.php]: Not doing that after all, is more in line with the other JIRA dev commands.

**Pull request merged**